### PR TITLE
Minio updates

### PR DESCRIPTION
**Pull request by maaikelimper**

### CODE REVIEW

File: docs/source/storage.rst [hunk 1]
```diff
@@ -35,6 +35,61 @@ which is received by wis2box.
...
+
+To allow uploading files into MinIO remotely, the ``wis2box-incoming`` bucket is proxied via Nginx. 
+
+For example, to upload the local file (``WIGOS_0-454-2-AWSNAMITAMBO_2021-11-18T0955.csv with topic``) with topic ``data.core.observations-surface-land.mw.FWCL.landFixed`` via the Nbinx proxy:
+
+.. code-block:: python
+
+    import boto3
+
+    endpoint_url = 'http://localhost:9000'
```
Comment (webb-ben) [inline docs/source/storage.rst:77]: Should this be `endpoint_url = $WIS2BOX_URL/wis2box-incoming`?

File: docs/source/storage.rst [hunk 2]
```diff
@@ -46,14 +101,14 @@ Install the Python minio module via `pip`_:
...
-The below example copies a local file (``myfile.csv``) to the ``wis2box-incoming`` bucket to topic ``foo/bar/baz``:
+The below example copies a local file (``myfile.csv``) to the ``wis2box-incoming`` bucket to topic ``foo.bar.baz``:
 
 .. code-block:: python
 
     from minio import Minio
 
     client = Minio(
-        'localhost',
+        'localhost:3000',
```
Comment (webb-ben) [inline docs/source/storage.rst:111]: `localhost:9000`?

File: docker/nginx/nginx.conf [hunk 3]
```diff
@@ -28,6 +28,22 @@ http {
...
 
+        # Proxy requests to the bucket "wis2box-incoming" to MinIO container running on port 9000
+        # NOTE do not use rewrite, it crashes the upload
+        location /wis2box-incoming/ {
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-Forwarded-Proto $scheme;
+            proxy_set_header Host $http_host;
+
+            proxy_connect_timeout 300;
```
Comment (webb-ben) [inline docker/nginx/nginx.conf:39]: Do we need this long of a timeout?

**Review by webb-ben — CHANGES_REQUESTED** [review]

File: docker/nginx/nginx.conf (hunk 3 above)
Comment (maaikelimper) [reply docker/nginx/nginx.conf:39]: not sure, this was taken from the example https://docs.min.io/docs/setup-nginx-proxy-with-minio

File: docs/source/storage.rst (hunk 1 above)
Comment (maaikelimper) [reply docs/source/storage.rst:77]: wis2box-incoming is the bucket and is defined when calling upload_fileobj. I will replace this with <your-wis2box-url>

File: docs/source/storage.rst (hunk 2 above)
Comment (maaikelimper) [reply docs/source/storage.rst:111]: yes

**Review by webb-ben — APPROVED** [review]

**Pull request merged**